monsieurbiz/SyliusStuartDeliveryPlugin
# Send Stuart job when finish checkout

## How to test
- Switch to this branch
- `make install`
- Got to https://127.0.0.1:8001/admin/settings/edit/monsieurbiz.stuart_delivery 
- You could find the API credentials in Team vault in 1Password, in Stuart Sandbox
- You could use the following package address:
```
59 Rue du Faubourg Poissonnière
75009 Paris
```
- Choose a transport type, Bike for example
- Make an order with a Parisian address too. For example:
```
106 rue de Montreuil
75011 Paris
```
- Choose `Stuart Delivery` as shipping method and a slot
- Finish your order using any payment method
- Run ``
- Go to https://dashboard.sandbox.stuart.com/scheduled with credentials in Team vault in 1Password, in Stuart Sandbox
- You should see something like this:
<img width="1317" alt="image" src="https://user-images.githubusercontent.com/872996/172606698-c3006333-d4f9-4e88-a391-f76347531765.png">
- You also could check delivery status in your customer account:
<img width="1153" alt="image" src="https://user-images.githubusercontent.com/872996/172612584-77dfbe6f-e2cc-44e6-a812-de7a7f3c217b.png">
